azure-webapp-maven-plugin/README.md

```diff
@@ -55,6 +55,7 @@ Maven | 3.0 or above
                     <schemaVersion>v2</schemaVersion>
                     <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
                     <appName>${WEBAPP_NAME}</appName>
+                    <region>${REGION}</region>
```

Comment:
`<region>` is not a required attributes with default value `westeurope`, shall we add it to quick start?

Reply:
No, without it, deploy will fail